src/test/java/com/pubnub/api/endpoints/access/GrantTokenEndpointTest.java

```diff
@@ -27,6 +29,27 @@ public void beforeEach() throws IOException {
         pubnub.getConfiguration().setSecretKey("secretKey").setIncludeInstanceIdentifier(true);
     }
 
+    @Test
+    public void test() throws PubNubException {
+        PNConfiguration conf = new PNConfiguration();
+        conf.setPublishKey("pub-c-395744cc-604e-410e-9074-ade176d5aac0")
+                .setSubscribeKey("sub-c-fdc9b0fc-3e1a-11ea-9e28-4eda21f1b082")
+                .setSecretKey("sec-c-M2YzM2MxMDgtM2EyNS00ZTkxLWEzM2ItM2E5MTJlYTc0NGFi")
+                .setOrigin("ingress-tcp-pub.az1.pdx1.aws.int.ps.pn")
```

Comment:
hardcoded keys and origin.  Perhaps better to replace them with environment variable names or something else.

Reply:
yeah. That shouldn't be here. Thanks